### PR TITLE
Tmp call

### PR DESCRIPTION
**Pull request by rodsol**
call the functions from the remaining  r2l helpers

### CODE REVIEW

File: opencog/nlp/relex2logic/rule-helpers.scm [hunk 1]
```diff
@@ -1128,6 +1184,8 @@
 	[else 
     (list (InheritanceLink (ConceptNode var1_instance) (ConceptNode var1))
 	(InheritanceLink (ConceptNode var2_instance) (ConceptNode var2))
```
Comment (AmeBel) [inline opencog/nlp/relex2logic/rule-helpers.scm]: correct the square brackets above

**Comment by williampma** [comment]
Do the non-instanced WordNode also need to link to the non-instanced ConceptNode/PredicateNode?  I guess that should be part of Word Sense Disambiguation.

**Comment by AmeBel** [comment]
@williampma correct the syntax error please thanks

**Comment by williampma** [comment]
where's the error? square brackets are allowed in scheme.

**Comment by AmeBel** [comment]
i didn't know that, `[ConceptNode "what"]` works.

**Pull request merged**